x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/index.tsx

```diff
@@ -38,6 +39,31 @@ export function ServiceGroupsList() {
     []
   );
 
+  const { start, end } = useMemo(
+    () =>
+      getDateRange({
+        rangeFrom: 'now-24h',
+        rangeTo: 'now',
+      }),
+    []
+  );
```

Comment:
This is on the service group overview page (aka the page listing all the service groups), right? 

@gbamparop and I talked about that we didn't currently have a timepicker, and we didn't see any need to add any. It also means that the lookback time-period for calculating the service counts is hardcoded to 24 hours. 

I think this is a fine compromise between simplicity and functionality. But we should remember to flag this to product (and possible docs to document it).

Reply:
Correct, this is on the service group overview page.